src/EVEMon.Common/Enumerations/PlanEntrySort.cs

```diff
@@ -21,6 +21,7 @@ public enum PlanEntrySort
         TimeDifference,
         PlanType,
         Notes,
-        SkillPointsRequired
+        SkillPointsRequired,
+        OmegeRequired
```

Comment:
Just a small thing, but there is a typo here

Reply:
LOL! Thanks, fixed!